src/types/index.ts

```diff
@@ -139,7 +139,9 @@ export type PlaidLinkOptionsWithPublicKey = CommonPlaidLinkOptions & {
 export type PlaidLinkOptionsWithLinkToken = CommonPlaidLinkOptions & {
   // Provide a link_token associated with your account. Create one
   // using the /link/token/create endpoint.
-  token: string;
+  // If token is null or undefined, the Link iframe will preloaded, but Link
+  // will not be initialized, and `ready` will be false
+  token?: string;
```

Comment:
This seems like an easy way our customers can shoot themselves in the foot...   a LinkTokenOptions with no token? I'd rather the beauty of a typing system kick in and say HEY DON'T DO THAT

Reply:
oops this was supposed to be in #197 not here... good catch